src/blocks/memory.rs

```diff
@@ -86,7 +86,7 @@ pub async fn run(config: &Config, api: &CommonApi) -> Result<()> {
         .await?;
 
     let mut format = config.format.with_default(
-        " $icon $mem_avail.eng(prefix:M)/$mem_total.eng(prefix:M)($mem_total_used_percents.eng(w:2)) ",
+        " $icon $mem_used.eng(prefix:Mi)/$mem_total.eng(prefix:Mi)($mem_used_percents.eng(w:2)) ",
```

Comment:
Just wanted to double check, do you think using the mem_ued and base 2 units is the way to go?

I tried to do some research about what is used (base 2 or 10) and I believe RAM and disk size are usually reported with base 2 units while network speed is measured in base 10 units.

Reply:
Probably. At least, `htop` does so.